extra-resources/mods/Mythos/common.cardset

```diff
@@ -118,9 +145,38 @@ card('Healer') {
         }
     }
 }
-
+// https://upload.wikimedia.org/wikipedia/commons/e/ec/Ricci_Guangqi_2.jpg
+// CC BY-SA 3.0
+// https://commons.wikimedia.org/wiki/User:SteveMcCluskey
```

Comment:
> 1602-1680

That must be the original creator. How they could license something under CC in the 17th century is beyond me.